client.py

```diff
@@ -145,6 +145,9 @@ def main():
                     continue
                 if "count" in s and s["count"] > limit: 
                     s["count"] = limit
+                if ("max_memory_per_job" in config_cluster 
+                    and s["memory"] >= config_cluster["max_memory_per_job"]):
```

Comment:
Shouldn't it be `>` rather than `>=`?  If we set it to 4GB and a 4GB request comes it that would be denied under this scheme.  But I think that should be accepted.